venia-integration-tests/src/fixtures/multiStore/data/storeB/product2ProductDetail.json

```diff
@@ -372,230 +372,6 @@
             }
           ],
           "variants": [
-            {
```

Comment:
instead of deleting this test data, shouldn't a new empty variant get created to be used to test against?

Reply:
This is just the removal of one variant. From what i can tell, when a product option is disabled, any variants that include that option are not included in the variant array.